mozilla/kuma
# Bug 1197444: Move JavaScript to kuma/static/js

Some general testing guidelines:

* Verify that JavaScript URLs still resolve
* Verify that the site still behaves as expected
* Verify that Intern tests still pass
